source/index.html

```diff
@@ -122,7 +122,11 @@ <h2 class="secondary-title example__title">Живой пример</h2>
                   </div>
                 </dl>
 
-                <span class="example__costs">Затраты на питание: 15 000 &#8381;</span>
+                <span class="example__costs">
+                  <span class="example__costs-text">
+                    Затраты на питание:
+                  </span> 15 000 &#8381;
+                </span>
```

Comment:
Как-то после закрытого тега текст плохо видно. Я вот не сразу понял зачем 2 обертки вокруг текста, не увидел его =)
Можно так:
```suggestion
                <span class="example__costs">
                  <span class="example__costs-text">
                    Затраты на питание:
                  </span>
                  15 000 &#8381;
                </span>
```

Или покомпактнее:
```suggestion
                <span class="example__costs">
                  <span class="example__costs-text">Затраты на питание:</span>
                  15 000 &#8381;
                </span>
```